chart/values.yaml

```diff
@@ -1,8 +1,79 @@
 ssmPath: /dsva-vagov/vets-api/dev/
 parameters:
-  SETTINGS__HOSTNAME: "hostname"
-  SETTINGS__VSP_ENVIRONMENT: "vsp_environment"
-  SETTINGS__VIRTUAL_HOSTS: "virtual_hosts"
```

Comment:
are these being deleted?

Reply:
Good catch @patrick-black-oddball. I readded the parameters